cli/src/bin/phylum.rs

```diff
@@ -63,6 +63,19 @@ async fn handle_commands() -> CommandResult {
     let ver = app.get_version().unwrap().to_owned();
     let matches = app.get_matches();
 
+    // Set the log level based on CLI arguments.
+    let log_level = match (matches.get_count("verbose"), matches.get_count("quiet")) {
+        (_, 2..) => "off",
+        (_, 1) => "error",
+        (0, 0) => "warn",
+        (1, _) => "info",
+        (2, _) => "debug",
+        (3.., _) => "trace",
+    };
+
+    // Initialize logger.
+    env_logger::Builder::from_env(Env::default().default_filter_or(log_level)).init();
```

Comment:
I'm wondering about this (admittedly obscure) case:

```
export RUST_LOG=trace
phylum -qq ping
```

I think a user would expect logging to be turned off for that call. But the code here would use the environment variable instead of the flags.

Reply:
Hm I could make the CLI take precedence, sure.